src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUserResponse.java

```diff
@@ -2,23 +2,34 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.util.StringUtils;
 import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUser;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUserStatus;
 
 public class ProfessionalUserResponse {
```

Comment:
Please use ProfessionalUsersResponse directly as this has been used already in other endpoint. Remove/Delete this java file.